Tests/E2E/README.md

```diff
@@ -18,17 +18,47 @@ This guide helps you to execute and author E2E tests on your local environment.
  +---------+
 ```
 
-* LoRaWan solution up and running (IoT Edge Device, IoT Hub, LoRa Keys Azure Function, Redis, etc.)
+* LoRaWan solution up and running (IoT Edge Device, IoT Hub, LoRa Keys Azure Function, Redis, container registry etc.)
 * Seeeduino LoRaWan device (leaf test device) connected via USB to a computer where the LoRaWan.Tests.E2E will run.
 * Module LoRaWanNetworkSrvModule logging configured with following environment variables:
   * LOG_LEVEL: 1
   * LOG_TO_UDP: true
   * LOG_TO_UDP_ADDRESS: development machine IP address (ensure IoT Edge machine can ping it)
 * E2E test configuration (in file `appsettings.local.json`) has UDP logging enabled `"UdpLog": "true"`
 
-## Installation
+## Setup
 
-1. Connect and setup Seeduino Arduino with the serial pass sketch
+### LoRa Basics Station and LNS setup
+
+Configure LoRa Basics Station and Network Server to run on your local LoRa gateway:
```

Comment:
By local LoRa gateway you mean what exactly? I don't think it's clear to the reader. I assume you mean the concentrator and network server running on your local machine?

Reply:
It means both components running on the concentrator. I will update this.